debian/rules

```diff
@@ -1,114 +1,50 @@
 #!/usr/bin/make -f
-# -*- makefile -*-
-# Sample debian/rules that uses debhelper.
-# GNU copyright 1997 to 1999 by Joey Hess.
 
 # Uncomment this to turn on verbose mode.
 #export DH_VERBOSE=1
 
-CFLAGS = -Wall -Wno-format -g
-
 ifneq (,$(findstring noopt,$(DEB_BUILD_OPTIONS)))
-	CFLAGS += -O0
+	CFLAGS_MAINT += -O0
 else
-	CFLAGS += -O2
+	CFLAGS_MAINT += -O2
 endif
 ifeq (,$(findstring nostrip,$(DEB_BUILD_OPTIONS)))
 	INSTALL_PROGRAM += -s
 endif
 ifneq (,$(findstring thumb,$(DEB_BUILD_OPTIONS)))
-        CFLAGS += -mthumb
+	CFLAGS_MAINT += -mthumb
 endif
 ifeq (,$(findstring nolauncher,$(DEB_BUILD_OPTIONS)))
 	CONFIGURE_OPTIONS += --enable-maemo-launcher
 	USE_MAEMO_LAUNCHER = true
 endif
 
-CFLAGS += -DTAP_AND_HOLD
-
-CXXFLAGS = $(CFLAGS)
-
-config.status:
-	if [ ! -x configure ]; then ./autogen.sh; fi
-	dh_testdir
-	# Add here commands to configure the package.
-	CFLAGS="$(CFLAGS)" CXXFLAGS="$(CXXFLAGS)" ./configure --prefix=/usr --sysconfdir=/etc --mandir=\$${prefix}/share/man --infodir=\$${prefix}/share/info --disable-static $(CONFIGURE_OPTIONS)
-
-
-build: build-stamp
+CONFIGURE_OPTIONS += --disable-static
 
-build-stamp:  config.status
-	dh_testdir
+CPPFLAGS_MAINT += -DTAP_AND_HOLD
+CXXFLAGS_MAINT += -Wall -Wno-format -Wno-format-security -g
+LDFLAGS_MAINT += -Wl,--as-needed
 
-	# Add here commands to compile the package.
-	$(MAKE)
+export DEB_CPPFLAGS_MAINT_APPEND = $(CPPFLAGS_MAINT)
+export DEB_CXXFLAGS_MAINT_APPEND = $(CXXFLAGS_MAINT)
+export DEB_CFLAGS_MAINT_APPEND   = $(CFLAGS_MAINT)
+export DEB_LDFLAGS_MAINT_APPEND  = $(LDFLAGS_MAINT)
 
-	touch build-stamp
+%:
+	dh $@
 
-clean:
-	dh_testdir
-	dh_testroot
-	rm -f build-stamp 
+override_dh_auto_configure:
+	dh_auto_configure -- $(CONFIGURE_OPTIONS) --libexecdir=/usr/libexec
 
-	# Add here commands to clean up after the build process.
-	-$(MAKE) distclean
-
-	dh_clean 
-
-install: build
-	dh_testdir
-	dh_testroot
-	dh_clean -k 
-	dh_installdirs
-
-        # Add here commands to install the package into debian/hildon-application-manager.
-	$(MAKE) install DESTDIR=$(CURDIR)/debian/hildon-application-manager
+override_dh_install:
+	dh_install
+	dh_installxsession -u 'post 40'
+	if test "x$(USE_MAEMO_LAUNCHER)" = "xtrue"; then \
+		dh_maemolauncher -v; \
```

Comment:
Indentation of this line differs from the following "for"

Reply:
Fixed.